ilan/push_all

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#
+###########################################################################
+# (C) Copyright IBM Corporation 2015, 2016.                               #
+#                                                                         #
+# Licensed under the Apache License, Version 2.0 (the "License");         #
+# you may not use this file except in compliance with the License.        #
+# You may obtain a copy of the License at                                 #
+#                                                                         #
+#      http://www.apache.org/licenses/LICENSE-2.0                         #
+#                                                                         #
+# Unless required by applicable law or agreed to in writing, software     #
+# distributed under the License is distributed on an "AS IS" BASIS,       #
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.#
+# See the License for the specific language governing permissions and     #
+# limitations under the License.                                          #
+###########################################################################
+
+REPO=$1
+
+tmp=$(cat versions.csv| cut -d, -f1)
+
+versions=$(echo  $tmp | cut -d " " -f 2-)
```

Comment:
As for build_all: Not sure what this does - does $tmp not have the right information?